validator/testcases/regex.py

```diff
@@ -582,6 +582,23 @@ def tests(self):
             compat_type='warning')
 
 
+@register_generator
+class Gecko56RegexTests(CompatRegexTestHelper):
+    """Regex tests for Firefox 56 updates."""
+
+    VERSION = FX56_DEFINITION
+
+    def tests(self):
+        yield self.get_test_bug(
+            981796,
+            r'\bshowModalDialog\b',
```

Comment:
I tried to make this less prone to false positives with proper tree traversal but somehow didn't manage to get it being raised properly… I figured, we had regex tests for other functions like `onButtonClick` too so this might work.